### DISCUSSION TITLE
Mathematicists find new glasses

### DISCUSSION THREAD

**Discussion by budrap00** [a-cosmology-group/acg#63] [Cosmology Models]
https://www.livescience.com/physics-mathematics/dark-energy/the-expansion-of-the-universe-could-be-a-mirage-new-theoretical-study-suggests

**Comment by RedshiftDrift** [comment]
I still don't understand how someone can believe that a **metric transformation** can produce a different physical solution.

In any case, a *mass evolution model* is nothing new.  Arp and others worked on similar ideas.  The problem is that one can pick any suitable function for the mass as a function of time, fit distance data and get the required 'metric' of space.  But this fitting exercise does not provide any explanation for the other anomalies in astronomy, so the model is no more falsifiable than LCDM.

Like other models that are this specific, this one will be forgotten tomorrow.

**Reply by HanDeBruijn** [reply]
@marmetl .

_In any case, a mass evolution model is nothing new. __Arp and others__ worked on similar ideas._

Yes, that's why the other(s) jump in here :-)

_Like other models that are this specific, this one will be forgotten tomorrow._

You shall not be surprised that the other(s) disagree.

_The problem is that one can pick any suitable function for the mass as a function of time, fit distance data and get the required 'metric' of space._

**That is definitely not true.** Let me try to explain.
I shall present you _A brief Theory of Time_. I'm pretty sure you can fill in details where necessary.

**Hypotheses non fingo.** Our theory obviously starts with the assumption that the [__Variable Mass Theory__](https://www.youtube.com/watch?v=T7jVviVBzKs) is true.
And thus _intrinsic redshift_ is according to

$$
\large 1+z = \frac{m_0}{m}
$$

where $m=$ lighter mass in the past, $m_0=$ heavier mass here and now ( = $t_0$ ).

The ~~second assumption is that the~~ change of (elementary particle rest) mass $m$ is _uniform_. That is: it does not depend opon the origin $(0)$ of (true atomic) time $t$. We are taking limits in the end, with $\Delta t = (t-t_0)/N$ .

$$
\frac{m(\Delta t)}{m_0} = \frac{m(2\Delta t)}{m(\Delta t)} = \frac{m(3\Delta t)}{m(2\Delta t)} = \cdots \quad \Longrightarrow \quad
\lim_{N\to\infty} \frac{m(N\Delta t)}{m_0} = \left[\frac{m(\Delta t)}{m_0}\right]^N \quad \Longrightarrow \quad
\frac{m}{m_0}(t) = e^{k(t-t_0)}
$$

With $k$ some constant. Comparison with _intrinsic redshift_ $z$ (first assumption !) learns that $k$ must be equal to the Hubble paraneter.

$$
\Large \frac{m}{m_0} =  e^{H(t-t_0)}
$$

At the same time, the modern _second_ is defined by [quote: ] _the time that elapses during 9,192,631,770 cycles of the radiation produced by the transition between two levels of the cesium-133 atom_. But this radiation will be intrinsically redshifted as well, resulting in a different magnitude of one cycle elswehere in the cosmos, name it $\Delta t$ . Far away atomic clocks are running slower, because we are looking into the past.

$$
\frac{\Delta t}{\Delta t_0} = \frac{m_0}{m} =  e^{-H(t-t_0)} = e^{H(t_0-t)}
$$

On the other hand, we had the good old definition of the seconds as a [Fraction of an ephemeris year](https://en.wikipedia.org/wiki/Second#Fraction_of_an_ephemeris_year), defined by a fixed number of seconds (31 556 926) in a year, together with certain restrictions on the "here and now". Question is: what would happen if we had adopted this "orbital time" as the true time, combined with our Variable Mass Theory. Let's take a look at Orbital mechanics, especially [Elliptical orbits](https://en.wikipedia.org/wiki/Orbital_mechanics#Elliptical_orbits). Under standard assumptions the [orbital period](https://en.wikipedia.org/wiki/Orbital_period) $\Delta T$ of a body traveling along an elliptic orbit can be computed as:

$$
\Delta T = 2\pi\sqrt{\frac{a^3}{\mu}} \quad \mbox{with} \quad \mu = GM
$$

Of interest to us is _only_ the mass $M=m$. We see that there is a "redshift" for Orbtital clocks as well, which is however _different_ from the one for Atomic clocks, namely

$$
\frac{\Delta T}{\Delta T_0} = \sqrt{\frac{m_0}{m}}
$$

The two clocks are always _synchronized_ in such a way that $t_0=T_0$ and $\Delta t_0=\Delta T_0$ . Then it is easy to see that, when taking limits:

$$
\frac{\Delta T / \Delta T_0}{\Delta t / \Delta t_0} = \frac{\sqrt{m_0/m}}{m_0/m} \quad \Longrightarrow \quad \sqrt{\frac{m}{m_0}} = \frac{dT}{dt} = e^{H/2(t-t_0)}
$$

We have a differential equation for $T$ as a function of $t$ which can be easily integrated. To make a long story short:

$$
\frac{m}{m_0} = \left[1+\frac{H}{2}(T-T_0)\right]^2
$$

This means that we have arrived at the $m = at^2$ law by Arp and Narlikar, _without any resort to General Relativity !_

And no, we have no choice! One can __not__ pick any suitable function for the mass as a function of time. And there are two of them which are equal. I think we have seen this before (-: #39).

$$
\left[1+\frac{1}{2}H(T-T_0)\right]^2 = \frac{m}{m_0} = \Large e^{H(t-t_0)}
$$

**Reply by RedshiftDrift** [reply]
That's not the kind of metric transformation explained in the paper.

As discussed with you elsewhere, the "variable mass theory"
1. is based on the "ephemeris year", which has been measured to be unstable and not suitable for the purpose of establishing a time scale over billions of years, (c.f. [the solar system is unstable](http://www.scholarpedia.org/article/Stability_of_the_solar_system#Marginal_stability_of_the_Solar_System.))
2. requires at least two assumptions ([STz requires none](https://github.com/orgs/a-cosmology-group/discussions/39#discussioncomment-6258031)).

**Reply by HanDeBruijn** [reply]
_That's not the kind of metric transformation explained in the paper._

I don't care about papers that are reinventing the wheel. Two more points:
1. Carnot's idealized machines were _not suitable for the purpose of_ building a real combustion engine that is working for years.
Yet they have been imensely valuable for developing the Classical Thermodyamics theory. I'm just doing something analogous.
2. STz uses as many assumptions as UAC does. The only difference between us is that _you_ don't make them explicit.
My first assumption is that VMT is responsible for the redshift. Your first assumption is that STz is responsible for the redshift.
My second "assumption" is just the statement that the past is not different from the future, which is also implicit in your theory.

This is not a fair debate.

### Notes.

For someone with only a hammer, everything looks like a nail. And:
1. for someone with only Gravity, everything looks like a Black Hole
2. for someone with only [Plasma](http://plasmauniverse.info/), everything looks like a Z-pinch
3. for someone with only [SED](https://en.wikipedia.org/wiki/Stochastic_electrodynamics), everything looks like a [ZPE](https://en.wikipedia.org/wiki/Zero-point_energy)
4. for someone with only Optics, everything looks like a Laser

And, oh yes, before I forget: the __Leap Second__ has enjoyed a [revival](http://www.alternatievewiskunde.nl/cosmology/simpel.htm#ERR).

And [the solar system is unstable](http://www.scholarpedia.org/article/Stability_of_the_solar_system#Marginal_stability_of_the_Solar_System.) ? That's laughable ! Mankind is far more unstable at the moment than the whole solar system ! By the way, the Wikipedia page says: _marginally stable_. Which is different from _unstable_.

**Reply by ftessier** [reply]
Has time symmetry breaking not been observed in particle physics? In which case the past would be different from the future. Ethan Siegel does a good job of explaining the gist of that experiment:
 https://bigthink.com/starts-with-a-bang/laws-physics-not-time-symmetric/

**Reply by HanDeBruijn** [reply]
Name me something that has __not__ been "observed" in particle physics :-(
Or did you mean that time has an arrow? In that case the last part of [this webpage](http://www.alternatievewiskunde.nl/cosmology/klokken.htm) might be instructive.
Yes, I see it in the article now. But you don't need particle physics to become convinced of one-directional time.

**Reply by budrap00** [reply]
Fractionally charged particle (quarks), W, Z, Higgs have never been directly detected only inferred by peering at the rubble of LHC proton collisions through the special lens of a theory that requires them. Time is not a physically interacting, causal thing. There are no empirical observations that support that concept - none. 

Time does not have a "direction" any more than distance or space or temperature have a direction - they are all relational concepts that describe the behavior of and relationships between physical systems. Thermodynamic processes have a direction but it is not linear and reversible, it is cyclical. To the extent that the "laws" of thermodynamics are reversible they are demonstrably wrong. This becomes obvious if you step outside the closed system of 19th century laboratory physics into your backyard.

All clocks represent some form of repetitive physical process and all define distinct "time intervals" and none of those intervals is absolute. Nor are any of those intervals causally related to the physical processes they are a measure of. Relational time is the only thing that the empirical (scientific} evidence supports. 

If someone has mathematical or metaphysical reasons for believing in the existence of a causally interactive time (or space or spacetime) they are welcome to them, but those beliefs have no scientific basis. Science isn't supposed to traffic in unsubstantiated beliefs, only empirical facts. To the extent that modern theoretical physics is a hothouse of unsubstantiated beliefs it has become a hamster wheel of circular logic going nowhere fast. Reifying the relational concepts of space and time was a colossal error of 20th century physics. Einstein was right, Wheeler and his cohort were catastrophically wrong.

**Reply by ftessier** [reply]
> Relational time is the only thing that the empirical (scientific} evidence supports. (...)
> Reifying the relational concepts of space and time was a colossal error of 20th century physics.

I agree. I would argue further that "time" as we experience it is not fundamental, only relations between "events" are. Could our "time" be an emergent property?

**Reply by ftessier** [reply]
@HanDeBruijn

> Or did you mean that time has an arrow? In that case the last part of [this webpage](http://www.alternatievewiskunde.nl/cosmology/klokken.htm) might be instructive.

I thought it was very interesting that one could even come up with an attempt to test $T$ symmetry quite directly with an experiment. Clever!

The last part of the webpage you mention is based on the _very_ widespread **misconception** that entropy means "disorder", and that therefore structure implies a reduction of entropy. That idea has been popularized notably by Schrödinger in his otherwise excellent 1944 book [_What is Life?_](https://en.wikipedia.org/wiki/What_Is_Life%3F), this idea that "life is a fight against entropy."

Entropy is related to the number of possible states, not what we conceive of as "order". Imagine a closed box containing bouncing balls in wild motion. Eventually the balls naturally come to rest at the bottom of the box: a more "ordered" state. How can that be? Well, by transferring possible states from degrees of freedom we can detect, to degrees of freedom we can't (thus entropy is perspectival). Complexity is very much entropically favoured, otherwise we would not see it everywhere in cosmological structures, let alone the molecules of life!

**Reply by HanDeBruijn** [reply]
Hmm, I've always thought that _Information_ (: Claude Shannon) is the opposite of [Entropy](https://en.wikipedia.org/wiki/Entropy_(information_theory)) . Has this knowledge become rusty ?

**Reply by ftessier** [reply]
No, that is indeed a very robust definition of entropy. There is "bijection" between the concept of information (Shannon entropy) and number of possible states (Gibbs entropy). Both can be expressed as $S = - \sum \rho \log \rho$.

**Reply by budrap00** [reply]
> I would argue further that "time" as we experience it is not fundamental, only relations between "events" are. Could our "time" be an emergent property?

I don't really understand your point. Time describes a relationship between entities and events in the same way that distance and temperature do.  Relational concepts can't be more fundamental than the entities and events whose behavior those relational concepts describe, can they? Are distance and temperature emergent properties or simply the descriptive term for certain observed relational behaviors ?

**Reply by ftessier** [reply]
_(At the risk of veering off topic for this forum)_

For perfect clarity I mean that, ultimately, fundamental equations of physics ought not contain a `t` variable which corresponds to the concept of time (at our scale). Same goes for `x` corresponding to our concept of distance. These phenomena should themselves "emerge" from more fundamental concepts, just like temperature "emerges" from an underlying, more fundamental concept of velocity distribution.

For example, the [Wheeler-DeWitt equation](https://en.wikipedia.org/wiki/Wheeler%E2%80%93DeWitt_equation) essentially does away with time entirely, so my idea is at least not completely outrageous—nor original by the same token.

Of course, I have no clue how to do this, otherwise I would do it! 🤔 It has always appeared to me that space `x` and time `t` are so rigidly taken for granted in physics (even with the _flexibility_ afforded by relativity). Are these anthropocentric notions the next one from which physics shall break free?

**Reply by HanDeBruijn** [reply]
@ftessier . _Are these anthropocentric notions the next one from which physics shall break free?_ 
Now read the last lines of [Support by Hoyle](http://www.alternatievewiskunde.nl/cosmology/Hoyle.htm#TF) at my website.

**Comment by HanDeBruijn** [comment]
Extremely disappointing writeups. The paper itself starts with the usual bombastic prose. And proceeds with the common over-the-top GR _mathemacist_ (yep!) garbage. Skimming through all those not-interesting Einsteinian and Minskowskian formulas, we finally arrive at a section which is potentially interesting: __Applications__. That's easy enough: not a single one of the kind can be observed in the text.
So let's jump immediately to  __Conclusions__.

1. _Rather than due to an expanding Cosmos, observed redshifts to distant galaxies can be interpreted as the evolution of particle masses after conformal transformation of the FLRW metric into static or Minkowski space._ Very nice, but all this and more has already been published in a [1992 paper](https://articles.adsabs.harvard.edu/cgi-bin/nph-iarticle_query?1993ApJ...405...51N&defaultprint=YES&filetype=.pdf) by Jayant Narlikar and Halton Arp. Which by the way is not found in the references. (What's worse, __General Relativity is not needed at all__ for a derivation of the most relevant results).

As for the rest, I'm not interested in: _gravitational waves_ , _multiverse_ , _as an axion and could act as dark matter or dark energy_. Period.
